amy/workshops/urls.py

```diff
@@ -4,86 +4,171 @@
 
 urlpatterns = [
     # utility views
-    path('log/', views.changes_log, name='changes_log'),
-    path('version/<int:version_id>/', views.object_changes, name='object_changes'),
-    path('workshop_staff/', views.workshop_staff, name='workshop_staff'),
-    path('workshop_staff/csv/', views.workshop_staff_csv, name='workshop_staff_csv'),
-
+    path("log/", views.changes_log, name="changes_log"),
```

Comment:
I guess the GH linter workflow didn't allow you to leave the file styling untouched? Unfortunately I didn't fix it in time to ignore `urls.py`.

But on the other hand, it's just easier to keep no exceptions from Black, and the formatted file is still quite readable.

Reply:
@pbanaszkiewicz that's right. I couldn't pass the linter it without making an exception for this file.